frontend/src/components/SessionTimer.js

```diff
@@ -0,0 +1,224 @@
+import styled from 'styled-components/macro';
+import { useState } from 'react';
+import ButtonBig from './ButtonBig';
+import ButtonTab from './ButtonTab';
+import { GiMeeple } from 'react-icons/all';
+
+export default function SessionTimer({
+    setStartDate,
+    setSessionDuration,
+    session,
+}) {
+    const [timer, setTimer] = useState({ s: 0, m: 0, h: 0 });
+    const [startTimeForSession, setStartTimeForSession] = useState('');
+    const [stopTimeForSession, setStopTimeForSession] = useState('');
+    const [intervalId, setIntervalId] = useState();
+    const [gameHasStarted, setGameHasStarted] = useState(false);
+    const [duration, setDuration] = useState();
+
+    const onClickToStart = () => {
+        let startDate = new Date();
```

Comment:
let?

Reply:
apparently, I'm not quite sure about the difference, yet ^^'